terraform/install.sh

```diff
@@ -50,6 +50,30 @@ promptForBillingAccount() {
     exit 1;
   fi
 
+ #check for permission to link account
+  found_accounts=()
+  IFS=$'\n'
+  found_all_accounts=($found_all_accounts)
+  for acc in ${found_all_accounts[@]}
+  do
+    IFS=$'\t'
+    acc=($acc)
+    IFS=$'\n'
+    test_perm=$(gcloud beta billing accounts get-iam-policy "${acc[1]}" --flatten=bindings --filter="bindings.role:roles/billing.projectManager OR bindings.role:roles/billing.user OR bindings.role:roles/billing.admin" --format='value(bindings.members)')
```

Comment:
it seems that the [required roles](https://cloud.google.com/billing/docs/how-to/modify-project#required-permissions-change) to link a project (assuming a user is the owner) and a billing account are:
`roles/billing.admin` **OR** (`roles/billing.user` **+** `roles/billing.viewer`)

Checking for `roles/billing.projectManager` is not required because the projects in use are owned by the user.

Also, it is possible that the caller is not in the list (`bindings.members`). Please, make sure that the authenticated identity is in the list when you validate the billing permissions. You can use the following command to acquire the identity:
```bash
gcloud auth list --filter="status:active" --format="value(account)"
```